backend/chromeApiServer/src/test/java/com/canvas/service/helperServices/CanvasClientServiceTest.java

```diff
@@ -19,20 +19,25 @@
 import org.mockito.Captor;
 import org.mockito.Mock;
 import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.env.Environment;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
```

Comment:
This import is never used, please remove

Reply:
will do